src/cron/dailyDigest.ts

```diff
@@ -8,12 +8,17 @@ import {
   User,
   UserPersonalizedDigest,
   UserPersonalizedDigestSendType,
+  UserPersonalizedDigestType,
 } from '../entity';
 import { Cron } from './cron';
 import { isWeekend, addHours, startOfHour, subDays } from 'date-fns';
 import { DEFAULT_TIMEZONE } from '../types';
 
 const sendType = UserPersonalizedDigestSendType.workdays;
+const digestTypes = [
+  UserPersonalizedDigestType.Digest,
+  UserPersonalizedDigestType.ReadingReminder,
+];
```

Comment:
This is pretty neat and flexible moving forward to move between crons.
Isn't moving the reading reminder just moving this to the other one?

Reply:
Technically it is 😬
But I want to do it in a different PR nonetheless just to test it properly and ensure nothing is borked by it 😄